Configs/en-de_bt_secondary.yml

```diff
@@ -30,11 +30,11 @@ early_stop_start_time: 2048
 val_dataset_script: BackTranslation/wmt19/wmt19.py
 lang_pair: de-en
 models_shared: true
-output_dir: ./Output/en-de_pipeline
+output_dir: Output/en-de_pipeline/bt_sec
 
 # data config
 # backtranslation language
-data_dir: ./DataLink/cc
+data_dir: ./Data/cc
```

Comment:
Should this remain `./DataLink/cc`? To my knowledge there is no `./Data/cc/en_500k.txt` uploaded to Github so this code should complain IIUC

Reply:
Yeah, I guess you are right. Question: how is this DataLink thing working locally?  Do I miss any steps?

Reply:
ln -s <source> DataLink